src/components/Configure/content/fields/FieldMappings/FieldMappingRow.tsx

```diff
@@ -127,7 +127,7 @@ export function FieldMappingRow({
           marginBottom: '.25rem',
         }}
         >
-          <span style={{ fontWeight: 500 }}>{field.mapToDisplayName}</span>
+          <span style={{ fontWeight: 500 }}>{field.mapToDisplayName ?? field.mapToName}</span>
```

Comment:
fixes the bug where if field only has mapToName but not mapToDisplayName (which is a valid thing to do in amp.yaml), then it looks like this:
<img width="544" alt="Screenshot 2025-04-16 at 10 39 23 PM" src="https://github.com/user-attachments/assets/57b51a17-1879-46b8-adcc-3b7ffaa9e064" />